examples/stable_diffusion_xl/gm/modules/embedders/modules.py

```diff
@@ -145,9 +146,7 @@ def embedding(self, *tokens, force_zero_embeddings=None):
 
         if force_zero_embeddings is None:
             force_zero_embeddings = ()
-        for i in range(len(self.embedders)):
-            embedder = self.embedders[i]
-            token = tokens[i]
+        for embedder, token in zip(self.embedders, tokens):
```

Comment:
zip may not functional well in graph mode, be care when use it

Reply:
Indeed, one of the reasons why graph mode doesn't work is due to `zip`. Thanks!